Client/mods/deathmatch/logic/CClientColModel.cpp

```diff
@@ -1,21 +1,21 @@
 /*****************************************************************************
  *
- *  PROJECT:     Multi Theft Auto v1.0
- *               (Shared logic for modifications)
+ *  PROJECT:     Multi Theft Auto
  *  LICENSE:     See LICENSE in the top level directory
- *  FILE:        mods/shared_logic/CClientColModel.cpp
+ *  FILE:        mods/deathmatch/logic/CClientColModel.cpp
  *  PURPOSE:     Model collision (.col file) entity class
  *
  *****************************************************************************/
 
 #include "StdInc.h"
+#include "../Client/game_sa/CModelInfoSA.h"
+#include "../Client/game_sa/CColModelSA.h"
 
-CClientColModel::CClientColModel(CClientManager* pManager, ElementID ID) : ClassInit(this), CClientEntity(ID)
+CClientColModel::CClientColModel(CClientManager* pManager, ElementID ID) : ClassInit(this), CClientEntity(ID), m_pColModel(nullptr)
```

Comment:
> ` m_pColModel(nullptr)`

Setting the default in the class declaration is preferred (`m_pColModel = nullptr;`)